### PR TITLE
Fixes #12623: API : policies update

### PR DESCRIPTION
**Pull request by Haypierre**
https://www.rudder-project.org/redmine/issues/12623

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
Can you rebase your pull request with rudder-dev rebase please ?

**Comment by Haypierre** [comment]
PR rebased

File: rudder-core/src/main/scala/com/normation/rudder/services/ClearCacheService.scala [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    S.clearCurrentNotices
+    val modId = ModificationId(uuidGen.newUuid)
+
+    //clear agentRun cache
+    clearableCache.foreach {_.clearCache}
+
+    //clear node configuration cache
+    (for {
+      set <- clearNodeConfigurationCache(storeEvent = false, actor)
+      _ <-  eventLogRepository.saveEventLog(
```
Comment (VinceMacBuche) [inline rudder-core/src/main/scala/com/normation/rudder/services/ClearCacheService.scala]: Can you indent it correctly please ? exactly like it was done before https://github.com/Normation/rudder/pull/1946/files#diff-a778e7d11c9c87410a1c06ef983dbc53L120

**Comment by Haypierre** [comment]
Commit modified

**Comment by Haypierre** [comment]
Commit modified

**Pull request merged**